### PR TITLE
19 create flight control module

### LINKED ISSUE #7: set up continuous integration pipeline

### LINKED ISSUE #19: create flight control module
Create a module which will act as the flight computer. It should do the following:

- [x] Set up the connection to the game and load all celestial bodies, reference frames, vessels, and important parts

- [x] Track event sequences

- [x] Hand off control to modules in charge of each event sequence

- [x] End cleanly after splashdown

### PR DESCRIPTION
**Pull request by mdbernard**
Adds a skeleton structure to the codebase.
Creates commander.py, which will govern the flight (closes #19).
Creates a Continuous Integration pipeline (closes #7).
Adds a unit test directory with an example unit test using pytest, and actual unit tests for command.check_status.

### CODE REVIEW

File: src/sequence_controllers/sc_00_COM_ES1_mission_setup.py
Comment (jdc-pub) [inline src/sequence_controllers/sc_00_COM_ES1_mission_setup.py:28]: I'm not familiar with Kerbal - should we check for an `else` condition here? Is it possible that some other item (another planet in the Kerbal solar system perhaps) ends up going through this function?

I leave that to your judgement. Either way, this has my approval for merge.

**Review by jdc-pub — APPROVED** [review]

File: src/sequence_controllers/sc_00_COM_ES1_mission_setup.py
Comment (mdbernard) [reply src/sequence_controllers/sc_00_COM_ES1_mission_setup.py:28]: No, unfortunately there's not a better way to get these objects stored in short variables. There's a limited number of uniquely named bodies (< 30), so this loop will take like 0.1 seconds. Plus it's not running more than once.

**Pull request merged**
